Gemfile

```diff
@@ -155,6 +155,7 @@ group :replication, :manageiq_default do
 end
 
 group :rest_api, :manageiq_default do
+  gem "manageiq-api", :path => "../manageiq-api"
```

Comment:
Which part, the bundler groups?  @bdunne is going to have to deal with it... the order is "specific" group first.  😉